helix-rest/src/test/java/org/apache/helix/rest/server/TestClusterAccessor.java

```diff
@@ -283,7 +284,8 @@ public void testEnableDisableMaintenanceMode() throws IOException {
     System.out.println("Start test :" + TestHelper.getTestMethodName());
     String cluster = _clusters.iterator().next();
     String reason = "Test reason";
-    // enable maintenance mode
+    // enable maintenance modecHelixController  - Exception while executing DEFAULT pipeline: RestSuperClusterActivationTest_SuperCluster for cluster [org.apache.helix.manager.zk.zookeeper.ZkClient.acquireEventLock(ZkClient.java:1103), org.apache.helix.manager.zk.zookeeper.ZkClient.waitForKeeperState(ZkClient.java:1081), org.apache.helix.manager.zk.zookeeper.ZkClient.waitUntilConnected(ZkClient.java:1070), org.apache.helix.manager.zk.ZKHelixManager.checkConnected(ZKHelixManager.java:372), org.apache.helix.manager.zk.ZKHelixManager.getSessionId(ZKHelixManager.java:823), org.apache.helix.controller.stages.MessageGenerationPhase.createStateTransitionMessage(MessageGenerationPhase.java:354), org.apache.helix.controller.stages.MessageGenerationPhase.generateMessage(MessageGenerationPhase.java:235), org.apache.helix.controller.stages.MessageGenerationPhase.processEvent(MessageGenerationPhase.java:98), org.apache.helix.controller.stages.resource.ResourceMessageGenerationPhase.process(ResourceMessageGenerationPhase.java:35), org.apache.helix.controller.pipeline.Pipeline.handle(Pipeline.java:68), org.apache.helix.controller.GenericHelixController.handleEvent(GenericHelixController.java:640), org.apache.helix.controller.GenericHelixController.access$400(GenericHelixController.java:117), org.apache.helix.controller.GenericHelixController$ClusterEventProcessor.run(GenericHelixController.java:1168)]. Will not continue to next pipeline
```

Comment:
Is that an accidental pasting?

Reply:
Good catch. Right, I will remove the later part.